src/components/MeetupList.js

```diff
@@ -262,6 +262,12 @@ const meetups = [
     emoji: ":thailand:",
     location: "Chiang Mai",
     link: "https://www.facebook.com/groups/219236462407862/",
+  },  
+  {
+    title: "Web3 Together (Phnom Penh)",
```

Comment:
No need to duplicate the location in the title, it will be displayed with the location tag :-)

```suggestion
    title: "Web3 Together",
```